docs/geozarr-eopf-specification.qmd

```diff
@@ -0,0 +1,304 @@
+---
+title: "GeoZarr for EOPF - Experimental Specification"
+subtitle: "Transforming Earth Observation Processing Framework Zarr to GeoZarr V3"
+author: "GeoZarr Examples Project"
+date: today
+format:
+  html:
+    toc: true
+    toc-depth: 3
+    code-fold: false
+    code-tools: true
+---
+
+## Overview
+
+This document outlines the experimental GeoZarr specification for transforming EOPF (Earth Observation Processing Framework) Zarr stores into GeoZarr V3 compliant datasets. The specification is based on the implementation in `06_multiscales_as_WebMercatorQuad_EOPFZarrV3.ipynb` and demonstrates how GeoZarr could be adapted for Zarr format 3 while maintaining compatibility with existing geospatial tools.
+
+GeoZarr V3 extends the Zarr V3 format to support geospatial data with proper coordinate reference systems, multiscale overviews, and CF-compliant metadata, specifically addressing the gaps found in EOPF Zarr format.
+
+## Gaps in EOPF Zarr Format
+
+The current EOPF Zarr stores have several gaps that prevent them from being fully GeoZarr compliant:
+
+### Issues Identified
+
+| Issue | Description | Impact |
+|-------|-------------|---------|
+| **Missing CF Standard Names** | Variables lack `standard_name` attributes | Reduces interoperability with CF-compliant tools |
+| **Incomplete CRS Information** | CRS stored only as `proj:epsg` attribute | Limited compatibility with rioxarray and geospatial tools |
+| **Missing Grid Mapping** | No `grid_mapping` attribute linking to spatial reference | Geospatial tools can't detect coordinate system |
+| **No Multiscale Support** | Lacks overview levels and multiscale metadata | Poor performance for multi-scale visualization |
+| **Missing Coordinate Arrays** | Overview levels lack proper x/y coordinate arrays | Cannot perform spatial operations on overview data |
+| **Inconsistent Geotransform** | Geotransform not in CF-compliant format | Spatial referencing may not work across tools |
+
+## GeoZarr V3 for EOPF - Core Requirements
+
+### 1. CF Convention Compliance
+
+All GeoZarr V3 datasets MUST follow CF (Climate and Forecast) conventions:
+
+```json
+{
+  "standard_name": "toa_bidirectional_reflectance",
+  "grid_mapping": "spatial_ref"
+}
+```
+
+**Requirements:**
+- All data variables MUST have a `standard_name` attribute
+- All geospatial data variables MUST reference a spatial reference system via `grid_mapping`
+
+### 2. Enhanced Spatial Reference System
+
+Every GeoZarr V3 dataset MUST include a `spatial_ref` coordinate variable with comprehensive CRS information:
+
+```json
+{
+  "spatial_ref": {
+    "attrs": {
+      "crs_wkt": "PROJCS[\"WGS 84 / UTM zone 32N\"...]",
+      "spatial_ref": "PROJCS[\"WGS 84 / UTM zone 32N\"...]", 
```

Comment:
```suggestion
```
I think only `crs_wkt` should be included because `spatial_ref` is not found as an attribute in CF compliant NetCDF files or as a tag in GeoTIFFs, it's a redundancy added by rioxarray for interoperability with GDAL in memory. IMO it's best not to persist those redundancies on disk.